molten.c

```diff
@@ -113,7 +109,11 @@ static const zend_module_dep molten_deps[] = {
     ZEND_MOD_OPTIONAL("redis")    
     ZEND_MOD_OPTIONAL("mongodb")    
     ZEND_MOD_OPTIONAL("mysqli")    
+#ifdef HAS_PDO
+    ZEND_MOD_REQUIRED("PDO")
```

Comment:
I make it "required" to avoid (again) undefined symbol:
`PHP Warning:  PHP Startup: Unable to load dynamic library 'modules/molten.so' - modules/molten.so: undefined symbol: php_pdo_get_exception in Unknown on line 0
`